pandas/core/interchange/column.py

```diff
@@ -194,6 +213,12 @@ def describe_null(self):
             column_null_dtype = ColumnNullType.USE_BYTEMASK
             null_value = 1
             return column_null_dtype, null_value
+        if isinstance(self._col.dtype, ArrowDtype):
+            # We already rechunk (if necessary / allowed) upon initialization, so this
```

Comment:
It comes from here:

https://github.com/pandas-dev/pandas/blob/d85c904a0e1120ba645006889a2b281a2ef9c36c/pandas/core/interchange/dataframe.py#L31-L41

In this PR I'm doing the rechunking (if allowed) upfront, so that when it's necessary to make a buffer for each column, we can just use `.chunks[0]`

Reply:
Ah OK thanks for the clarification on that. And is the reason for rechunking that the dataframe interchange protocol does not support chunks?

Reply:
yes, that's right:

> A Buffer class. A buffer is a contiguous block of memory - this is the only thing that actually maps to a 1-D array in a sense that it could be converted to NumPy, CuPy, et al. 

https://data-apis.org/dataframe-protocol/latest/API.html